testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/environment/EnvSinglenodeSparkIceberg.java

```diff
@@ -57,20 +59,25 @@ public EnvSinglenodeSparkIceberg(Standard standard, Hadoop hadoop, DockerFiles d
     @Override
     public void extendEnvironment(Environment.Builder builder)
     {
+        String dockerImageName = "ghcr.io/trinodb/testing/hdp3.1-hive:" + hadoopImagesVersion;
+
         builder.configureContainer(HADOOP, container -> {
-            container.setDockerImageName("ghcr.io/trinodb/testing/hdp3.1-hive:" + hadoopImagesVersion);
+            container.setDockerImageName(dockerImageName);
             container.withCopyFileToContainer(
                     forHostPath(dockerFiles.getDockerFilesHostPath("conf/environment/singlenode-spark-iceberg/apply-hive-config-for-iceberg.sh")),
                     CONTAINER_HADOOP_INIT_D + "/apply-hive-config-for-iceberg.sh");
         });
 
+        builder.configureContainer(TESTS, dockerContainer -> {
+            dockerContainer.withCopyFileToContainer(
+                    forHostPath(dockerFiles.getDockerFilesHostPath("conf/tempto/tempto-configuration-for-hive3.yaml")),
+                    CONTAINER_TEMPTO_PROFILE_CONFIG);
```

Comment:
CONTAINER_TEMPTO_PROFILE_CONFIG is deprecated. what's the current way to accomplish this?

Reply:
It lists `EnvironmentContainers.configureTempto` as its replacement, but that method doesn't work in this case (because that only works when the tempto configuration has a specific name and is in the environment's config directory).

Quite a few things around the environment configuration need to be refactored, and I avoided changing it as much as possible.

(This is also copied verbatim from `EnvSinglenodeHdp3`, so if we change it here, we should probably also change it there.)

Reply:
so CONTAINER_TEMPTO_PROFILE_CONFIG is deprecated but we cannot use the replacement?
was it deprecated too early? cc @kokosing

Reply:
The replacement works in most cases, but there are a few where it doesn't. To address that, we should probably add an overload like `configureTempto(Environment.Builder, String)` to specify which file to use rather than taking a file from a `ResourceProvider`.